app/Nova/DocuSignEnvelope.php

```diff
@@ -99,9 +100,29 @@ public function fields(NovaRequest $request)
             Text::make('DocuSign Envelope ID', 'envelope_id')
                 ->onlyOnDetail(),
 
-            URL::make('DocuSign URL', 'url')
+            URL::make('View in DocuSign', 'url')
+                ->displayUsing(static fn () => 'Signer View')
```

Comment:
What does this do?

Reply:
Sets the label for the link - see test

Reply:
https://nova.laravel.com/docs/4.0/resources/fields.html#url-field

Reply:
![image](https://user-images.githubusercontent.com/10482109/174387462-313ab9f1-8a0e-44ef-83a3-31627ae39b5e.png)

I don't particularly like this UX, but, it's fine.